connectors/sources/mysql.py

```diff
@@ -31,7 +31,7 @@
 SPLIT_BY_COMMA_OUTSIDE_BACKTICKS_PATTERN = re.compile(r"`(?:[^`]|``)+`|\w+")
 
 MAX_POOL_SIZE = 10
-DEFAULT_FETCH_SIZE = 50
+DEFAULT_FETCH_SIZE = 5000
```

Comment:
Initially 50 was the fetch size from the database used in the fetchmany() function now we have implemented custom pagination and are using fetchall() instead of fetchmany() hence we have increased the fetch size to 5000 (it is a configurable field in kibana), during testing we found 5000 to be effective.